### ISSUE TITLE
navigator version is incorrectly detected in Facebook's android webview.

### ISSUE THREAD

**Issue by buggedcom** [madferro/Sniff.js#4]
The user agent for that is 

`Mozilla/5.0 (Linux; Android 8.0.0; TA-1004 Build/OPR6.170623.013; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/62.0.3202.84 Mobile Safari/537.36 [FB_IAB/FB4A;FBAV/151.0.0.44.205;]`

producing 

```
sniff {
    version: "1.3",
    browserVersionExtended: "4.0",
    mobile: ["Android"],
    browserType: "chrome",
    browserVersion: 4
}
```

vs chrome actual

`Mozilla/5.0 (Linux; Android 8.0.0; TA-1004 Build/OPR6.170623.013) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.84 Mobile Safari/537.36`

```
sniff {
    version: "1.3",
    browserVersionExtended: "62.0.3202.94",
    mobile: ["Android"],
    browserType: "chrome",
    browserVersion: 62
}
```

**Comment by buggedcom** [comment]
```
function sniff(){this.version="1.3",this.browserType,this.browserVersion,this.mobile,this.browserVersionExtended=null;for(var e,i=/((iPhone|iPod|iPad).*AppleWebKit(?!.*Version)|; wv)/i.test(navigator.userAgent),s=!1,t=navigator.userAgent.match(/Android/i)||navigator.userAgent.match(/BlackBerry/i)||navigator.userAgent.match(/iPhone|iPad|iPod/i)||navigator.userAgent.match(/Opera Mini/i)||navigator.userAgent.match(/IEMobile/i)||null,n=navigator.appName,o=navigator.userAgent,r=o.match(/(opera|chrome|safari|firefox|msie|trident|edge)\/?\s*(\.?\d+(\.\d+)*)/i),a=o.match(/(opera|chrome|safari|firefox|msie|trident|edge)\/?\s*(\.?\d+(\.\d+)*)/gi),d=0;d<a.length;d++)if(a[d].toLowerCase().indexOf("edge")>=0){var m=a[d].split("/"),f=m[0].toLowerCase(),l=m[1];s=!0,console.log("so qui")}if(!s){r&&!i&&null!=(e=o.match(/version\/([\.\d]+)/i))&&(r[2]=e[1]),r=r?[r[1],r[2]]:[n,navigator.appVersion,"-?"];var f=r[0].toLowerCase(),l=r[1];if("trident"==r[0].toLowerCase()){f="msie",console.log(o),l=o.match(/(rv)\:?(\.?\d+(\.\d+)*)/i);for(var c=0;c<l.length;c++)if(!isNaN(parseInt(l[c]))){l=l[c];break}}}this.webview=i,this.mobile=t,this.browserType=f,this.browserVersion=parseInt(l),this.browserVersionExtended=l}var sniff=new sniff;"undefined"!=typeof document.documentElement.classList?(document.documentElement.classList.add(sniff.browserType),document.documentElement.classList.add("v"+sniff.browserVersion),null!=sniff.mobile&&document.documentElement.classList.add(sniff.mobile.toString().toLowerCase()),sniff.webview&&document.documentElement.classList.add("webview")):(document.documentElement.className+=" "+sniff.browserType+" v"+sniff.browserVersion,null!=sniff.mobile&&(document.documentElement.className+=" "+sniff.mobile.toString().toLowerCase()),sniff.webview&&document.documentElement.classList.add("webview"));
```

fixed and added support for `webview`

**Comment by madferro** [comment]
Thank you very much!!! I immediately update it
